CMakeLists.txt

```diff
@@ -34,16 +34,16 @@ else()
 endif()
 
 find_package(octomap REQUIRED)
-find_package(catkin COMPONENTS cmake_modules visualization_msgs shape_msgs resource_retriever random_numbers eigen_stl_containers)
```

Comment:
I think cmake_modules is still required for indigo, that is why they have a fallback ``# Fallback to cmake_modules`` here http://wiki.ros.org/jade/Migration